spec/presenters/crime_application_presenter_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe CrimeApplicationPresenter do
-  subject { 
-    described_class.new(
-      CrimeApplication.new(
-        status: 'in_progress', 
-        created_at: DateTime.new(2022, 01, 12))
-    ) 
+  subject { described_class.new(crime_application) }
+
+  let(:crime_application) {
+    instance_double(CrimeApplication,
+                    id: 'a1234bcd-5dfb-4180-ae5e-91b0fbef468d',
```

Comment:
By using this fake UUID, the expectation a few lines below continues to pass as we take the first 6 chars and prepend LAA, making it still `LAA-a1234b` 😄